.github/workflows/getFailedTests.js

```diff
@@ -0,0 +1,96 @@
+const core = require("@actions/core");
+const { Command } = require("commander");
+const { GraphQLClient } = require("graphql-request");
+
+const program = new Command();
+const client = new GraphQLClient("https://dashboard.cypress.io/graphql");
+
+program
+  .name("Approve PR")
+  .description("Approve and merge PR if patch release")
+  .option("--dashboard_url <dashboard_url>", "Cypress dashboard url")
+  .action(async options => {
+    const data = await getTestsStatusAndId(options.dashboard_url);
+
+    let testsStatus = data.status;
+
+    if (testsStatus === "FAILED") {
+      const testCases = await getFailedTestCases(data.runId);
+      if (testCases.length >= 20) {
+        core.setOutput("notifySlack", "true");
+      }
+    } else if (testsStatus === "FAILED") {
+      core.setOutput("notifySlack", "true");
+    }
+  })
+  .parse();
+
+async function getTestsStatusAndId(dashboardUrl) {
+  const getProjectRegex = /\/projects\/([^\/]*)/;
+  const getRunRegex = /\/runs\/([^\/]*)/;
+
+  const requestVariables = {
+    projectId: dashboardUrl.match(getProjectRegex)[1],
+    buildNumber: dashboardUrl.match(getRunRegex)[1],
+  };
+
+  const throwErrorAfterTimeout = setTimeout(function () {
+    throw new Error("Run have still running status, after all tests executed");
+  }, 1200000);
+
+  const data = await waitForTestsToFinish(requestVariables);
+
+  clearTimeout(throwErrorAfterTimeout);
+  return { status: data.status, runId: data.id };
+}
```

Comment:
We already have such logic, could you create a module and reuse it instead? 
- https://github.com/saleor/saleor-dashboard/blob/607b5d40a5a69b2be11e3d6418cf16694fd6a077/.github/workflows/approveAndMergeReleasePR.js#L179-L204
- https://github.com/saleor/saleor-dashboard/blob/607b5d40a5a69b2be11e3d6418cf16694fd6a077/.github/workflows/approveAndMergeReleasePR.js#L136-L153